dcat/index.html

```diff
@@ -1482,7 +1494,427 @@ <h4>Property: is referenced by</h4>
 
         </section>
 
+<!-- Versioning properties -->
+
+<section id="Property:resource_previous_version">
+
+<h4>Property: previous version</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a href="http://www.w3.org/ns/dcat#previousVersion">dcat:previousVersion</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>The previous version of a resource in a lineage [[?PAV]].</td>
+</tr>
+<tr>
+<td class="prop">Equivalent property:</td>
+<td><a data-cite="?PAV#d4e459"><code title="http://purl.org/pav/previousVersion">pav:previousVersion</code></a></td>
```

Comment:
All the links to PAV return the RDF, not the web pages of the terms. Is that what we want?  For the other vocabularies, we point to the web page or a URI that is resolved in a web page.  
if no smarter trick applies, I would link to the GitHub page using href similarly to what we usually do for the DCT terms. 
```suggestion
<td><a href="https://pav-ontology.github.io/pav/#d4e459"><code title="http://purl.org/pav/previousVersion">pav:previousVersion</code></a></td>
```

Reply:
Thanks for spotting this bug, @riccardoAlbertoni . I fix it in a new commit, by changing the URL of the PAV bib entry in `config.js` - see https://github.com/w3c/dxwg/pull/1295/commits/4d1ff20ccc272e84befe7857aadb21bb7d08fc8d